### ISSUE TITLE
Old projects page to new projects

### ISSUE THREAD

**Issue by iparask** [radical-group/radical-group.github.io#18]
These are the projects from the old page:

### Federally Funded Active:
 * [x] RADICAL-Cybertools
 * [x] ExTASY Project
 * [x] NSF CAREER Award
 * [x] Imagery Cyberinfrastructure and Extensible Building-Blocks to Enhance Research in the Geosciences (ICEBERG)
 * [x] Middleware (MIDAS) and High Performance Analytics Libraries for Scalable Data Science (SPIDAL)  (An NSF DIBBS Project)
 * [x] The Power of Many: Ensemble Toolkit for Earth Sciences (Website coming)
 * [ ] Replica-Exchange on NSF Production Cyberinfrastructure (and here)
 * [x] Next-generation Workload Management Systems for High Energy and Nuclear Physics for Future Extreme Scale Scientific Applications. (Department of Energy project, Fall 2016-19)
 * [x] Abstractions and Integrative Middleware for Extreme Scales (AIMES) (Department of Energy)
 * [x] Polar Cyberinfrastructure and Polar Computing Research Coordination Network.
 * [ ] Streaming Systems
 * [x] Fast FingerPrinting and Material Detection: A DOE SBIR Project.
 * [ ] Cybermanufacturing: NSF EAGER and Industrial project with J&J (2015-2016)
 * [x] Blue Waters Petascale Resource Allocation (PRAC, 2017-19)
 * [x] Molecular Science Software Sustainability Institute (2016-2021)
 * [x] Campus Compute Co-operative (CCC)

### Recently Finished
 * [ ] SCIHM Project (2012-2015)
 * [ ] Blue Waters Petascale Resource Allocation (PRAC, 2015-16)
 * [ ] EXTENCI  (2011-2014)
 * [ ] Distributed Programming Abstractions (DPA) (2006-2009)
 * [ ] D3 Science (2008-2010)
 * [x] Cybertools
 * [ ] Joining the TeraGrid
 * [ ] Conceptualization of an NSF Institute for Biomolecular Simulation

### RADICAL Cybertools 
RADICAL members are responsible for a standards-based and an abstractions based approach to high-performance distributed computing, including SAGA -- its implementation, deployment and standardization efforts, as well as the many  associated projects in cyberinfrastructure research and development, computational science and computing research.
 * [x] SAGA-Python
 * [x] RADICAL-Pilot

### Cyberinfrastructure and Cyberinfrastructure-based Science & Engineering Projects
 * [ ] Extending National Cyberinfrastructure (ExTENCI)
 * [ ] Standards-based Cyberinfrastructure for Hydrometrogical Modeling (SCIHM)
 * [ ] DRIHMS2US
 * [ ] Large-Scale Replica-Exchange on National Production Cyberinfrastructure: An NSF Cyber-enabled Discovery and Innovation (CDI) Project
 * [x] ExTASY: Supporting Scalable Approaches to Large Ensembles - Molecular Simulations

### D3 Projects
 * [ ] D3 Science and Distributed Data-Intensive Science

### Projects on Distributed Computing Abstractions and Models

 * [ ] Distributed Programming Abstractions (DPA)
 * [ ] P *:  A Model for Pilot-Abstractions
 * [ ] TROY
 * [ ] Pilot-MapReduce (PMR) and Next-Generation Gene Sequencing

### Computational Science Projects
 * Computational Science/Biology Projects:
   - [ ] Nucleic Acid Transport through Protein Nanopore
   - [ ] Riboswitch

---

As you can see, I would like to avoid this clutter and keep two sections: __Active Projects__ and __Past Projects__ regardless if a project is funded or not.

@shantenujha can you please open tickets and assign people to them?

**Comment by iparask** [comment]
This ticket will remain open until everything has been marked

**Comment by shantenujha** [comment]
@iparask  -- Apologies, I'm not sure what I should open tickets for?  Can you elaborate, please? @

**Comment by iparask** [comment]
I am asking to open tickets for every project that should be in the page and assign the person who is responsible for the project to that ticket. But that may take too much time without any improvement.

So, I suggest that for every PR with a project, I mark one of these as resolved. The ticket will close when all are closed.

Just a reminder, I still do not have a page for MOLSSSI that will work as a template for the rest

**Comment by shantenujha** [comment]
Is this what you're looking for @iparask 
https://github.com/radical-group/radical-group.github.io/issues/25

**Comment by mturilli** [comment]
@shantenujha based on the current projects and #27, #28, and #29, what other projects should we add from the list above?

**Comment by iparask** [comment]
> Is this what you're looking for @iparask
> #25

Yes! That is what I was asking for.

**Comment by iparask** [comment]
> @iparask is this consistent with your mental model:
> http://radical.rutgers.edu/projects/molssi

Yes this is based on mine mental model, although we said that it would be based on yours!

**Comment by mturilli** [comment]
Can this be closed or should we add more projects to the project page?

**Comment by iparask** [comment]
If @shantenujha agrees, yes we can close it.

**Comment by shantenujha** [comment]
Please keep this open, as there are some "old" projects that I'd like to still transfer. Sorry for delay.

**Comment by iparask** [comment]
Is this issue still need to be open?

**Comment by shantenujha** [comment]
Lets leave this open, as there are a few more projects to add  and some to move into “former” category.

Shantenu


On Sep 30, 2019, at 2:16 PM, Ioannis Paraskevakos <notifications@github.com<mailto:notifications@github.com>> wrote:


Is this issue still need to be open?
